versioned_docs/version-6.x/getting-started.md

```diff
@@ -12,7 +12,7 @@ If you're already familiar with JavaScript, React and React Native, then you'll
 
 Here are some resources to help you out:
 
-1. [React Native Express](https://www.reactnative.express) (Sections 1 to 4)
+1. [React Native](https://reactnative.dev/docs/getting-started) (Sections 1 to 4)
```

Comment:
Why are we changing this reference?

Reply:
Yeah, I see that last commit in React Native Express was about 3 years ago so we can proceed with that, but we should remove `(Sections 1 to 4)` then 🤔